modules/openapi-generator/src/main/resources/Java/libraries/resttemplate/ApiClient.mustache

```diff
@@ -769,23 +770,32 @@ public class ApiClient{{#jsr310}} extends JavaTimeFormatter{{/jsr310}} {
         ResponseEntity<T> responseEntity = null;
         int attempts = 0;
         while (attempts < maxAttemptsForRetry) {
-            try {
-                responseEntity = restTemplate.exchange(requestEntity, returnType);
-                break;
-            } catch (HttpServerErrorException ex) {
-                attempts++;
-                if (attempts < maxAttemptsForRetry) {
-                    try {
-                        Thread.sleep(waitTimeMillis);
-                    } catch (InterruptedException e) {
-                        Thread.currentThread().interrupt();
-                    }
+          try {
```

Comment:
i wonder if you can indent this code (mustache) block with 4 spaces instead of 2.

Reply:
have done. thank you.
previously I used 'google java format' plugin in Intellij which used 2 spaces I think. and I see it is line with google style guide (which is recommended in CONTRIBUTING.md)
am I missing something?

Reply:
> google style guide (which is recommended in CONTRIBUTING.md) 

that's outdated. i'll update/fix it later